go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/nspcc-dev/go-ordered-json
+
+go 1.17
+
+require (
+	github.com/virtuald/go-ordered-json v0.0.0-20170621173500-b18e6e673d74
```

Comment:
Add `rewrite` to get it from `nspcc-dev/go-ordered-json`?

Reply:
Isn't `rewrite` should be added to neo-go?

Reply:
Fixed.